Engine/Module/ModuleAudio.h

```diff
@@ -38,6 +38,8 @@ class ModuleAudio : public Module
 	ENGINE_API void SetRTPCValue(const std::string& name, int value);
 	ENGINE_API void ResetRTPCValues(const std::string& name);
 
+	int volume = 100;
```

Comment:
you are not using this variable right?

Reply:
Yes it is used on the (for now) debug window to be shown as the actual volume. Now this variable is replaced on the WorldManager since it should be unique in the Game.

*Once the UI Settings is done this variable should be moved over there